CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.12.15+8
```

Comment:
ah, gotcha, interesting convention. So not necessarily intending to release right away. Makes sense.